src/linux/mem_reader.rs

```diff
@@ -1,7 +1,11 @@
 //! Functionality for reading a remote process's memory
 
-use super::{minidump_writer::MinidumpWriter, serializers::*, Pid};
+use {
+    super::{minidump_writer::MinidumpWriter, serializers::*, Pid},
+    std::sync::OnceLock,
+};
 
+#[derive(Debug)]
```

Comment:
Is the `Debug` trait required or can it be removed? I wouldn't add it unless the structure is user-printable.

Reply:
Yep -- This is needed to be able to use `.expect()` on the potential `Err(Style)` that might be returned by `OnceLock::set()`. Alternatively, I could use `.map(|_| ()).expect(...)` on it to avoid this, but honestly this is an internal type anyway - What's the point of avoiding `#[derive(Debug)]` on it?